### PR TITLE
DATAGO-67636: Update README.md to include CLI commands plus some general cleanup

### PR DESCRIPTION
**Pull request by gregmeldrum**
### What is the purpose of this change?

 Update the README.md to reflect the new CLI commands.

### How was this change implemented?

Added a new section to the README with an explanation of how to use it.

### How was this change tested?

Reviewed by @AndreaKelso 

### Is there anything the reviewers should focus on/be aware of?

No

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,9 +46,9 @@ The Event Management Agent was tested to run with
 * 1 CPU
 * 1 GB RAM
 
-## Prerequisites:
+## Prerequisites
 
-* Docker
```
Comment (gregmeldrum) [inline README.md:51]: @AndreaKelso Here is the rewording for the prerequisites section.

File: README.md [hunk 2]
```diff
@@ -180,10 +180,74 @@ curl -X 'GET' \
   --output ih9z9lwwv5r.zip
 ```
 
-# Manually uploading scans to Event Portal
+## Running a Discovery Scan and File Export Using the CLI
+
```
Comment (gregmeldrum) [inline README.md:184]: @AndreaKelso Added additional context for the CLI to make it clear the CLI can be accessed from the docker container or the jar file.

File: README.md [hunk 3]
```diff
@@ -180,10 +180,74 @@ curl -X 'GET' \
...
+* provide the name of the Event Management Agent container (e.g. event-managenement-agent)
+* provide the Event Management Agent Docker image (e.g. solace/event-management-agent:latest)
+* add all password environment variables to the environment of the docker container (example BROKER_PASSWORD)
+
+The commands are as follows:
+`CMD_LINE_ARGS="scan abcdefg123 /tmp/scan.zip  --springdoc.api-docs.enabled=false --spring.main.web-application-type=none"`
+`run -d -v /path/to/file/AcmeRetailStandalone.yml:/config/ema.yml -v /path/to/scandir:/tmp  --env CMD_LINE_ARGS=${CMD_LINE_ARGS} --env BROKER_PASSWORD=${BROKER_PASSWORD} --name event-management-agent event-management-agent:latest`
+
+## Running a Discovery File Upload Using the CLI
+
```
Comment (gregmeldrum) [inline README.md:217]: @AndreaKelso I was missing the word "upload" in the first sentence. Hopefully it's clear now.

File: README.md [hunk 4]
```diff
@@ -180,12 +180,76 @@ curl -X 'GET' \
...
+To run a scan file upload using the CLI from docker:
+* provide the path and filename of the scan data zip file. (e.g. /tmp/scan.zip)
+* provide the location of the connection file (e.g. /path/to/file/AcmeRetail.yml)
+* provide the name of the Event Management Agent container (e.g. event-managenement-agent)
+* provide the Event Management Agent Docker image (e.g. solace/event-management-agent:latest)
+
+The commands are as follows:
+`CMD_LINE_ARGS="upload /tmp/scan.zip  --springdoc.api-docs.enabled=false --spring.main.web-application-type=none"`
+`run -d -v /path/to/file/AcmeRetail.yml:/config/ema.yml -v /path/to/scan.zip:/tmp  --env CMD_LINE_ARGS=${CMD_LINE_ARGS} --name event-management-agent event-management-agent:latest`
```
Comment (moodiRealist) [inline README.md:245]: I think this should be `docker run -d ...` instead

**Review by moodiRealist — APPROVED** [review]
Added two minor comments, looks good otherwise!

File: README.md [hunk 5]
```diff
@@ -180,12 +180,84 @@ curl -X 'GET' \
...
+* provide the name of the Event Management Agent container (e.g. event-managenement-agent)
+* provide the Event Management Agent Docker image (e.g. solace/event-management-agent:latest)
+* add all password environment variables to the environment of the docker container (example BROKER_PASSWORD)
+
+The commands are as follows:
+
+```
+CMD_LINE_ARGS="scan mysolacebroker /path/to/scan.zip  --springdoc.api-docs.enabled=false --spring.main.web-application-type=none"
+docker run -d -v /path/to/file/AcmeRetailStandalone.yml:/config/ema.yml -v /path/to/scandir:/tmp  --env CMD_LINE_ARGS=${CMD_LINE_ARGS} --env BROKER_PASSWORD=${BROKER_PASSWORD} --name event-management-agent event-management-agent:latest
+```
```
[the remark this answers is not in the document]
Comment (moodiRealist) [reply README.md]: @gregmeldrum can you please use "```" instead? The new refactoring makes the code block read weird and doesn't have the copy to clipboard option anymore.
Screenshot 2024-02-01 at 1 16 28 PM

**Pull request merged**